### ISSUE TITLE
Playlist has empty names

### ISSUE THREAD

**Issue by gerroon** [xbmc/Kore#341]
Hi

I am playing some radios from an imported .m3u. I can see the currently playing song/playlist name info item ok, the actual playlist has proper number of the playlist items but with empty names. So say I have 20 radios in the playlist, I will 20  empty lines in the playlist section in Kore. Yatzee shows the names properly on the other hand.

My Kodi server is running on Debian x64 with Kodi 16.1 and I am using the latest Fdroid build for android.

**Comment by SyncedSynapse** [comment]
Hopefully c9147231abf98c4d1467853f46735ed2f6a1dcd1 fixes this, not sure though.

**Comment by gerroon** [comment]
Hi

I think that fixed it but I am having a bizare twist here which probably is not related Kore.


I use Kodi on Debian, so until the other day there was only Jaris avail. My m3u radio listings would show up fine in Kodi (except in Kore).


I just updated KOre to the latest and updated Kodi to Krypton which became available yesterday. Now the situation reversed ;( I can see file radio listing fine in Kore but every line in a  radio playlist in Kodi has plain 0s as file name. I tried m3u and psl without much luck. I know the radio names are properly stored in these files.
